src/test/java/touro/chess/RookPieceTest.java

```diff
@@ -0,0 +1,50 @@
+package touro.chess;
+
+import junit.framework.TestCase;
+import org.junit.Test;
+import java.util.List;
+import static org.junit.Assert.assertTrue;
+
+public class RookPieceTest
+{
+
+    @Test
+    public void getMoves00()
+    {
+        //given
+        RookPiece rook = new RookPiece(new Location(0,0), PieceColor.Black);
+        //when
+        List<Move> list = rook.getMoves();
+        //then
+        for(Move move: list)
+        {
+            assertTrue(move.getTo().getColumn()==0 ^ move.getTo().getRow()==0);
```

Comment:
explain what you are doing here with `^`

Reply:
I am making sure that the output is one but not the other but now that I look up the definition is this just checking that its not both but not checking one of them is 0. if so how would I check that one is 0 and not the other.

Reply:
Sure. But it's more straightforward and more correct to check the values of row and column.
As written, if *every* item in the list is (0,1) for instance, your test will still pass, even though it's wrong.